async-query/build.gradle

```diff
@@ -28,7 +28,7 @@ dependencies {
     implementation group: 'org.json', name: 'json', version: '20231013'
     api group: 'com.amazonaws', name: 'aws-java-sdk-emr', version: "${aws_java_sdk_version}"
     api group: 'com.amazonaws', name: 'aws-java-sdk-emrserverless', version: "${aws_java_sdk_version}"
-    implementation group: 'commons-io', name: 'commons-io', version: '2.8.0'
+    implementation group: 'commons-io', name: 'commons-io', version: '2.14.0'
```

Comment:
can we set it as a constant?

Reply:
i think it's pretty common practice that these dependency package just have version defined inline, telling from all other build.gradle files

Reply:
yes

Reply:
can you create a issue to track this? thx!

Reply:
sure thing